src/WebJobs.Script.WebHost/WebScriptHostExceptionHandler.cs

```diff
@@ -29,6 +29,13 @@ public async Task OnTimeoutExceptionAsync(ExceptionDispatchInfo exceptionInfo, T
         {
             FunctionTimeoutException timeoutException = exceptionInfo.SourceException as FunctionTimeoutException;
 
+            // this seems to happen when the worker channel is already shutting down. Ex. One timeout is being handled and another comes in during shutdown.
+            if (timeoutException is null)
+            {
+                _logger.LogDebug("A timeout exception has occurred, but worker channel is already shutting down", exceptionInfo.SourceException);
```

Comment:
So, this part of the message "_but worker channel is already shutting down_", we are logging that without actually checking the state of the worker channel but simply based on the exception here. Is that a reliable signal to make that conclusion? Should we update the message here to remove the channel state?